Libraries/LibWeb/Painting/ViewportPaintable.cpp

```diff
@@ -181,6 +181,10 @@ void ViewportPaintable::refresh_scroll_state()
 
     m_scroll_state.for_each_sticky_frame([&](auto& scroll_frame) {
         auto const& sticky_box = scroll_frame->paintable_box();
+
+        if (!sticky_box.sticky_insets_ptr())
+            return;
```

Comment:
Why is this the correct fix? Why are insets missing?

This looks like we're just papering over a symptom without addressing the root cause.

Also this kind of change needs a test indeed :)